okxchain/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "okxchain",
+  "assets": [
+    {
+      "description": "EVM compatible, Cosmos SDK based chain.",
+      "denom_units": [
+        {
+          "denom": "wei",
+          "exponent": 0
+        },
+        {
+          "denom": "okt",
+          "exponent": 18
+        }
+      ],
+      "base": "wei",
+      "name": "OKXChain",
+      "display": "OKT",
+      "symbol": "OKT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/okxchain/images/okc.svg",
```

Comment:
these images look more like images to be used for the chain, and if so, please use these images for the chain logo. Please instead point the asset's logo URIs to a token logo (would that be the Black OKX-square.png?).